pages/index.jsx

```diff
@@ -0,0 +1,26 @@
+import MainWrapper from "@/components/wrappers/MainWrapper";
+import Description from "@/components/sections/Description";
+import Header from "@/components/layout/Header";
+import Even from "@/components/sections/Even";
+import Head from "next/head";
+import Footer from "@/components/layout/Footer";
+import useScale from "@/hooks/useScale";
+
+export default function Home() {
+  const { windowWidth } = useScale();
+  return (
+    <>
+      <Head>
+        <title>Even</title>
+      </Head>
+      {windowWidth !== 0 && (
```

Comment:
why are you checking the `windowWidth` here

Reply:
We're checking the value of the windowWidth variable because we use it to calculate the scale of the text on the page. Since the initial value of windowWidth is zero on the first render and on page reload, it takes a few seconds for the browser to determine the actual size of the window. That is why we need to render once it is not zero for the scale to make effect.

Reply:
Ok great, makes sense